frontend/src/containers/DocumentList.js

```diff
@@ -459,7 +471,7 @@ class DocumentListContainer extends Component {
    * @method getData
    * @summary Loads view/included tab data from REST endpoint
    */
-  getData = (id, page, sortingQuery, locationAreaSearch) => {
+  getData = (id, page, sortingQuery, locationAreaSearch, websocketRefresh) => {
```

Comment:
too many params. Can we use object param in here ?

Reply:
Dude this is a friggin' hack I want to remove as soon as possible and not spend a second more on it than needed.